lib/Entities/NewsletterEntity.php

```diff
@@ -23,6 +23,7 @@ class NewsletterEntity {
   const TYPE_WELCOME = 'welcome';
   const TYPE_NOTIFICATION = 'notification';
   const TYPE_NOTIFICATION_HISTORY = 'notification_history';
+  const TYPE_WC_TRANSACTIONAL_EMAIL = 'wc_transactional';
```

Comment:
Correct me if I am wrong, but I thought we decided to use the full `WooCommerce` rather than using the `WC` abbreviation.